.github/workflows/pytest.yml

```diff
@@ -33,6 +33,12 @@ jobs:
       - name: Run tests
         run: ./bin/coverage.sh
 
+      - name: Upload coverage report
+        uses: actions/upload-artifact@v3
+        with:
+          name: coverage-report
+          path: coverage
```

Comment:
Should this match the path in [`bin/coverage.sh`](https://github.com/cal-itp/eligibility-api/blob/main/bin/coverage.sh#L11)?

Reply:
Yes, thank you for catching! Done in 70c2825